_posts/2019-02-07-checking-the-status-of-your-api-submitted-deal.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title:  "Checking the Status of Your API Submitted Deal"
+date:   2019-02-07 4:00:00 -0400
+categories: api
+author: Kyle Chan
+---
+
+*Now that you have submitted your first deal via our [Lead Submission Endpoint](http://tech.forwardfinancing.com/api/2017/03/07/using-our-lead-submissions-api.html),
+you're probably wondering how it's doing.*
+
+## Getting Started
+The only things you'll need are 
+- The **lead ID** you received when you first submitted your deal
+- Your **API key**
+
+#### Authentication
+
+You will need to contact us to get your **API key**. The API key should be included
+in your request headers such that the request header key is `api_key` and the
+request header value for that key is your **API key**. If you omit the api_key or
+provide an incorrect one, the HTTP status code of the response will be 401
+Unauthorized.
+
+#### HTTPS
+
+Since your API requests will contain sensitive personal information about our
+customers, you must use SSL (HTTPS). To discourage insecure API requests,
+the response code will be 403 Forbidden for all requests made over HTTP.
+
+#### Rate Limiting
+
+We impose a rate limit per user of 1 request per second. If you exceed this rate
+of requests, the response code will be 429 Too Many Requests. If you need to
+exceed this limit for some reason, please let us know!
+
+## The Deal Status Endpoint
+
+You will be able to check your deal status with a GET request to the `/v1/deal_status` 
+endpoint with your **lead ID**.
+For example, the GET url might be
+
+`https://api-staging.forwardfinancing.com/v1/deal_status/<LEAD_ID>`
+
+### Request
+
+The only thing you'll need in the request header is your **API key**!
+
+```
+api_key: YOUR_API_KEY_HERE
+```
+
+### Response
+The possible response body fields are 
+- stage
+- maxApproval
+- maxPayments
+- offerLink
+- declineDrivers
+- declineNotes
+- missingInfo
```

Comment:
If these are the returned attribute names (they should be) please format this as the actual JSON payload. It's also very unfortunate that we're using camalCase here I'd say, since as far as I can tell reading over the Lead Submission Endpoint docs, we use underscores everywhere else.